### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by breville**
Remove some unused code initially discovered by using Chrome's Coverage tab.

### CODE REVIEW

File: src/datasetManifest.js [hunk 1]
```diff
@@ -250,9 +250,3 @@ export function getAvailableDatasets(specificDatasets) {
   // Sort dataset names by alphabetical order.
   return availableDatasets.sort((a, b) => a.name.localeCompare(b.name));
 }
-
-export function getDefaultLabelForDataset(datasetId) {
-  return allDatasets.find(dataset => {
-    return dataset.id === datasetId;
-  });
-}
```
Comment (Erin007) [inline src/datasetManifest.js:258]: Hmmm -  there are levels in the curriculum where a default label is set (preference, for example, in the food datasets. How do we do that without this code?

File: src/redux.js [hunk 2]
```diff
@@ -92,20 +90,12 @@ export function setSelectedTrainer(selectedTrainer) {
   return { type: SET_SELECTED_TRAINER, selectedTrainer };
 }
 
-export function setKValue(kValue) {
-  return { type: SET_K_VALUE, kValue };
-}
```
Comment (Erin007) [inline src/redux.js:97]: I think we can also remove line 36 `const SET_K_VALUE = "SET_K_VALUE"; and then the kValue in initial state and also where we reference that state in the algorithm.  I'd even prefer if deleting `setKValue` and its tendrils lived in its own PR.

File: src/redux.js [hunk 3]
```diff
@@ -92,20 +90,12 @@ export function setSelectedTrainer(selectedTrainer) {
...
 export const setColumnsByDataType = (column, dataType) => ({
   type: SET_COLUMNS_BY_DATA_TYPE,
   column,
   dataType
 });
 
-export function setSelectedFeatures(selectedFeatures) {
-  return { type: SET_SELECTED_FEATURES, selectedFeatures };
-}
-
```
Comment (Erin007) [inline src/redux.js:108]: Similar to above, can you remove the action types references for each of these that you're deleting, please.

File: src/redux.js [hunk 4]
```diff
@@ -443,12 +421,6 @@ export default function rootReducer(state = initialState, action) {
       trainedModel: action.trainedModel
     };
   }
-  if (action.type === SET_TRAINED_MODEL_DETAILS) {
-    return {
-      ...state,
-      trainedModelDetails: action.trainedModelDetails
-    };
-  }
   if (action.type === SET_TRAINED_MODEL_DETAIL) {
```
Comment (Erin007) [inline src/redux.js:424]: Just clarifying, `setTrainedModelDetails` and `setSelectedFeatures` can both be removed because we edit the individual fields for those objects?

File: src/redux.js [hunk 5]
```diff
@@ -696,15 +658,6 @@ export function getOptionFrequencies(state, column) {
...
 
-export function getOptionFrequenciesByColumn(state) {
-  let optionFrequenciesByColumn = {};
-  getSelectedCategoricalColumns(state).map(
-    column =>
-      (optionFrequenciesByColumn[column] = getOptionFrequencies(state, column))
-  );
-  return optionFrequenciesByColumn;
-}
-
```
Comment (Erin007) [inline src/redux.js:707]: This can be deleted because we now show the bar charts instead of these numbers?

File: src/datasetManifest.js (hunk 1 above)
Comment (breville) [reply src/datasetManifest.js:258]: Looks like we set it here: https://github.com/code-dot-org/ml-playground/blob/main/src/jsonReaderWrapper.js#L38

File: src/redux.js (hunk 4 above)
Comment (breville) [reply src/redux.js:424]: It looks like we do still have actions that modify them, at https://github.com/code-dot-org/ml-playground/blob/0c27d7ee37550eda2b7fc11b7821cf217507bd1e/src/redux.js#L452 and https://github.com/code-dot-org/ml-playground/blob/0c27d7ee37550eda2b7fc11b7821cf217507bd1e/src/redux.js#L339-L356 respectively.

File: src/redux.js (hunk 3 above)
Comment (breville) [reply src/redux.js:108]: Good catch, will do.

File: src/redux.js (hunk 5 above)
Comment (breville) [reply src/redux.js:707]: I think so.  I discovered this uncalled code through Chrome's coverage feature, and then couldn't find any remaining calls to this function.

File: src/redux.js (hunk 2 above)
Comment (breville) [reply src/redux.js:97]: Prepped a standalone PR in https://github.com/code-dot-org/ml-playground/pull/146 which can be merged first.

File: src/redux.js (hunk 5 above)
Comment (breville) [reply src/redux.js:707]: It turns out that `getOptionFrequencies`, immediately above, takes a `column`, so it seems to serve the right purpose for `ColumnInspector`.

**Review by Erin007 — APPROVED** [review]
Awe - one of the other tested functions and it's time to say goodbye.  Thanks for the clean up!
